doc/classes/TextEdit.xml

```diff
@@ -404,10 +404,10 @@
 			If [code]true[/code], read-only mode is enabled. Existing text cannot be modified and new text cannot be added.
 		</member>
 		<member name="scroll_horizontal" type="int" setter="set_h_scroll" getter="get_h_scroll" default="0">
-			The current horizontal scroll value.
+			If there is a horizontal scrollbar this determines how many pixels from the left the text is.
 		</member>
 		<member name="scroll_vertical" type="float" setter="set_v_scroll" getter="get_v_scroll" default="0.0">
-			The current vertical scroll value.
+			If there is a vertical scrollbar this determines which line is at the top of the box, the first line is 0.
```

Comment:
Similar thing here, maybe something like: "If there is a vertical scrollbar this determines the current vertical scroll value in line numbers, starting at 0 for the top line."